### ISSUE TITLE
Deprecated functions after updating to HA 2021.12.0

### ISSUE THREAD

**Issue by AdmiralStipe** [cytech/Home-Assistant-wundergroundpws#80]
Updating to latest HA 2021.12.0 throws out a bunch of warnings on deprecated functions on start - see attached log. 
[wunderground.log](https://github.com/cytech/Home-Assistant-wundergroundpws/files/7698335/wunderground.log)

**Comment by XD36JD20** [comment]
I too am seeing this behavior.

**Comment by TTLucian** [comment]
Some errors here too:
```
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:549
First occurred: 01:57:13 (54 occurrences)
Last logged: 01:57:54

Entity sensor.wupws_precip_chance_3d (<class 'custom_components.wundergroundpws.sensor.WUndergroundSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.wupws_precip_chance_4d (<class 'custom_components.wundergroundpws.sensor.WUndergroundSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.wupws_precip_chance_5d (<class 'custom_components.wundergroundpws.sensor.WUndergroundSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.wupws_winddir (<class 'custom_components.wundergroundpws.sensor.WUndergroundSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.wupws_today_summary (<class 'custom_components.wundergroundpws.sensor.WUndergroundSensor'>) implements device_state_attributes. Please report it to the custom component author.
```
```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:103
First occurred: 01:57:48 (2 occurrences)
Last logged: 01:57:53

Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue to the custom component author for wundergroundpws using this method at custom_components/wundergroundpws/sensor.py, line 538: with async_timeout.timeout(10, loop=self._hass.loop):
Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue to the custom component author for wundergroundpws using this method at custom_components/wundergroundpws/sensor.py, line 549: with async_timeout.timeout(10, loop=self._hass.loop):
```

**Comment by cytech** [comment]
fixed in v0.8.1
